spec/fixtures/affiliates.yml

```diff
@@ -45,6 +45,7 @@ another_affiliate:
   display_name: Another Gov Site
   name: another.gov
   api_access_key: another_key
+  youtube_profiles: nps
```

Comment:
needed nps youtube to have 2 affiliates to test out .distinct (replacement for .uniq)